app/templates/index.html

```diff
@@ -88,6 +95,16 @@
                             <label for="config-near">Near: </label>
                             <input type="text" name="near" id="config-near" placeholder="City Name">
                         </div>
+                        <div class="config-div">
+                            <label for="config-theme">Theme: </label>
+                            <select name="theme" id="config-theme">
+                                {% for theme in themes %}
+                                <option value="{{ theme }}" {% if theme in config.theme %} selected {% endif %}>
+                                    {{ theme }}
```

Comment:
Another minor nitpick here, but just to keep styling consistent throughout the config options:
```suggestion
                                    {{ theme.capitalize() }}
```